library/elixir

```diff
@@ -1,14 +1,20 @@
-# maintainer: denc716@gmail.com (@c0b)
+# this file is generated via https://github.com/c0b/docker-elixir/blob/50bb23cfce41cafeb4f6bd0a1eeeb412ddc780ca/generate-stackbrew-library.sh
 
-1.2.5:  git://github.com/c0b/docker-elixir@22ee98417200ef8d9a049b2b4504e7cf279e911f 1.2
-1.2:    git://github.com/c0b/docker-elixir@22ee98417200ef8d9a049b2b4504e7cf279e911f 1.2
-latest: git://github.com/c0b/docker-elixir@22ee98417200ef8d9a049b2b4504e7cf279e911f 1.2
+Maintainers: Mr C0B <denc716@gmail.com> (@c0b)
+GitRepo: https://github.com/c0b/docker-elixir.git
```

Comment:
With the new 2822-based format, we only `git fetch` on `refs/head/master` by default, so this PR fails to find the referenced commit -- you'll either need to get those commits onto `master`, reference `master` commits, or add `GitFetch: refs/heads/use-precompiled-elixir-1.3`. :+1:

(Travis would've caught this if it'd run, but some intermittent GitHub 404 apparently stopped it from doing so :cry:)